src/utils/SiriusAdapter.cpp

```diff
@@ -328,11 +342,15 @@ class TOPPSiriusAdapter :
                    << "--candidates" << QString::number(candidates)
                    << "--ppm-max" << ppm_max
                    << "--compound-timeout" << compound_timeout
-                   << "--tree-timeout" << tree_timeout 
+                   << "--tree-timeout" << tree_timeout
                    << "--quiet"
                    << "--output" << out_dir.toQString(); //internal output folder for temporary SIRIUS output file storage
 
-    // add flags
+    // check OpenMP
+    #ifdef _OPENMP
```

Comment:
I dont understand this, this is highly surprising for users. If OpenMS was built without OpenMP support, then you cannot have multithreading in java? I dont really understand the logic of this, why not simply pass forward the `--threads` parameter?

Reply:
Ok, I will do that.
So it does not matter if more threads are "called" than available - or is that not possible?

Reply:
Ok, put in the case of SIRIUS you will specify the number of processors with the thread parameter.